docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
       DATABASE_URL: postgres://postgres:mysecretpassword@db:5432/postgres
       YARN_INTEGRITY_ENABLED: "false"
     volumes:
-      - .:/usr/src/app
+      - .:/myusr/src/app
```

Comment:
Is this correct? We're still using `/usr/src/app` as our working directory.

Reply:
you're right fixed.